src/common.ts

```diff
@@ -13,17 +11,17 @@ export const BRIEF_DESCRIPTION =
 export const FULL_DESCRIPTION =
   'Compare GitHub changelogs across multiple releases in a single view'
 
-export const HIGH_PRIORITY_GROUP_TITLES: Array<string> = [
-  SemVerGroupTitles.breakingChanges,
-  SemVerGroupTitles.features,
-  SemVerGroupTitles.bugFixes,
+export const HIGH_PRIORITY_GROUP_TITLES = [
+  'breaking changes',
+  'features',
+  'bug fixes',
 ]
```

Comment:
This is a nice change! Easier to read _and_ takes up less space in the code.

Reply:
And the most important: less prone to errors!